src/KubernetesClient.csproj

```diff
@@ -11,6 +11,10 @@
 
     <TargetFramework>netstandard1.4</TargetFramework>
     <RootNamespace>k8s</RootNamespace>
+    <Version>1.1.0</Version>
```

Comment:
what those properties for?

Reply:
We've created a NuGet package with those changes and I have let it there by mistake. 
Fixed